src/common/utils.ts

```diff
@@ -13,14 +13,22 @@ import {
 } from "./interfaces/common.interface";
 import { ScientificRelation } from "./scientific-relation.enum";
 
+// add Å to mathjs accepted units as equivalent to angstrom
+const isAlphaOriginal = Unit.isValidAlpha
+Unit.isValidAlpha = function (c) {
+  return isAlphaOriginal(c) ||  c == 'Å'
+}
+createUnit('Å', '1 angstrom')
```

Comment:
This works since mathjs is only imported in a single file. However if we need to handle units from multiple files then we should refactor this so that the module setup code only gets called once.